### ISSUE TITLE
SEO Crawl Issues

### ISSUE THREAD

**Issue by jgeofil** [jgeofil/personal-web#28]
It looks like the action items are:

* View crawl results
* View 3XX redirect issues
* View missing alt text issues
* View meta description too short issues
* View redirected page has no incoming internal links issue
* View page has links to redirect issue
* View noindex page issue
* View nofollow page issue
* View meta description tag missing or empty issue
* View low word count issue
* View X (Twitter) card missing issue
